server/manifestmergetool/src/main/java/com/defold/manifestmergetool/AndroidManifestMerger.java

```diff
@@ -0,0 +1,80 @@
+package com.defold.manifestmergetool;
```

Comment:
Move this to a separate file. We have a separate merge class for HTML and plists so why not one AndroidManifest.xml?